src/fft.jl

```diff
@@ -106,9 +107,13 @@ function determine_grid_size(lattice::AbstractMatrix{T}, Ecut; supersampling=2,
     if ensure_smallprimes
         Vec3([nextprod([2, 3, 5], 2gs + 1) for gs in Gmax])
     else
-        Vec3([2gs+1 for gs in Gmax])
+        Vec3([2gs + 1 for gs in Gmax])
     end
 end
+function determine_grid_size(lattice::AbstractMatrix{T}, Ecut;
+                             kwargs...) where T <: Interval
+    determine_grid_size(mid.(lattice), Ecut; kwargs...)
+end
```

Comment:
Why is this needed?

Reply:
Because `ceil(Int, ::Interval{Float64})` cannot be properly implemented (`Interval{Int}` does not exist). I'll add a comment.

Reply:
I think the right thing to do here is to define ceil in IntervalArithmetic to ceil when possible and error it cannot be unambiguously defined

Reply:
What I want to avoid is "polluting" our code with this. We can avoid having the main code depend on intervals by widening the FFT code and defining the methods we need in a script

Reply:
Hmm ... that's probably fair, but I figured we'll probably depend on Intervals anyway if we start to use them more seriously. Also, if we want to test our code with Intervals, that it's at least a test dependency (though not an install one) :). I'll check how difficult it is to get this into IntervalArithmetic directly.

Reply:
We can isolate that in a test, while it's not in the main IntervalArithmetic. Better to pirate in a test than in the main code. Also not having to load it would help the loading times. I agree it's a test dependency though; can we express that? Ie depend on something only for tests?

Reply:
Sure ... test-only dependencies are easy. That's done for `DoubleFloats` already.